src/components/Dialog/index.test.js

```diff
@@ -23,6 +20,8 @@ describe('Dialog', () => {
       />
     );
 
+    expect(screen.getByLabelText('Close')).toHaveFocus();
```

Comment:
Going to start doing a lot more focus management & focus testing in these tests.